### PR TITLE
[Improve](GEO)wkb input and output are represented as hexadecimal strings And delete EWKB

### PR DESCRIPTION
**Pull request by LemonLiTree**
1. 
mysql> select ST_AsBinary(st_point(24.7, 56.7));
+----------------------------------------------+
| st_asbinary(st_point(24.7, 56.7))            |
+----------------------------------------------+
| \x01010000003333333333b338409a99999999594c40 |
+----------------------------------------------+
1 row in set (0.01 sec)
2.
Since the S2 library is not based on the WGS-84 coordinate system, it is not correct to construct EWKB format data in 4326, so delete

### CODE REVIEW

**Comment by LemonLiTree** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 32.49 seconds
 stream load tsv:          437 seconds loaded 74807831229 Bytes, about 163 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230417043436_clickbench_pr_129902.html

File: be/src/geo/wkb_parse.cpp [hunk 1]
```diff
@@ -43,6 +87,36 @@ GeoParseStatus WkbParse::parse_wkb(std::istream& is, bool isEwkb, GeoShape** sha
     return ctx.parse_status;
 }
 
+WkbParseContext* WkbParse::read_hex(std::istream& is, WkbParseContext* ctx) {
+    // setup input/output stream
+    std::stringstream os(std::ios_base::binary | std::ios_base::in | std::ios_base::out);
```
[the remark this answers is not in the document]
Comment (LemonLiTree) [reply be/src/geo/wkb_parse.cpp:92]: done

**Comment by LemonLiTree** [comment]
run buildall

**Comment by LemonLiTree** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]
LGTM

**Pull request merged**